CONTRIBUTING.md

```diff
@@ -0,0 +1,29 @@
+# Contributing to Forgerock's Javascript SDK
+
+:+1::tada: First off, thanks for taking the time to contribute! :tada::+1:
+
+The following is a set of guidelines for contributing to Forgerock and its packages, which are hosted in the [Forgerock](https://github.com/forgerock) on GitHub.
+
+### Git Flow
+
+Please fork the repository to your own github account. From there, you can add the forgerock repository as an upstream.
+
+`git remote add upstream git@github.com:ForgeRock/forgerock-javascript-sdk.git`
+
+Here is a checklist of items to ensure your Pull Request is reviewed and ready to be merged:
+
+- [ ] Please make all pull requests against `develop` branch.
+- [ ] Fill out the Pull Request template
+- [ ] When your PR is ready to merge please rebase your commit into 1 commit.
+- [ ] Commits should be verified (meaning you cannot squash your commit on github).
+- [ ] Please ensure your code is passing CI in order to get a review
```

Comment:
We may want to be a bit more specific here:

```md
- [ ] Please run `npm run lint` against your altered codebase
- [ ] Please run `npm run test` against your altered codebase
- [ ] Please ensure your PR passes all PR checks
```

Reply:
Please add periods '.' to the end of the sentences.

Reply:
I'm not used to using punctuation within short lists. Is this is an explicit content guideline? I personally prefer no terminators for list like this.

Reply:
According to the FR style guide, we are to use periods in bulleted lists:
https://wikis.forgerock.org/confluence/display/DOC/ForgeRock+Style+Guide+for+Technical+Publications

Jodie, the documentation manager told me today that the company will start being more stringent regarding documentation meeting the FR style guide conventions.

I would also suggest we put the word 'Please' at the top just once, then follow with list:
- Run `npm run lint` against your altered codebase
- Run `npm run test` against your altered codebase
- Ensure your PR passes all PR checks

Reply:
Okay, thanks, Sam. We will add them.